### PR TITLE
RN Proton + R-7 Avionics

### PR DESCRIPTION
**Pull request by Asgarmagol**
Updated the Proton Avionics to be able to control the craft.

### CODE REVIEW

**Comment by Asgarmagol** [comment]
Added R-7 configs too

File: GameData/RP-0/Avionics.cfg [hunk 1]
```diff
@@ -347,3 +347,51 @@
...
+		massLimit = 700.0
+	}
+}
+// Config for RN Proton (Block D-M2)
+@PART[rn_proton_blockd]:FOR[RP-0]
+{
+	MODULE
+	{
+		name = ModuleAvionics
+		massLimit = 20.0
```
Comment (NathanKell) [inline GameData/RP-0/Avionics.cfg]: @Asgarmagol is 20t really enough for the Blok D as well as the L3 complex?

**Comment by NathanKell** [comment]
Looks great, the exception being the line note I added--since Blok D was also used for TLI of the N1-L3 complex it probably  needs more avionics?

File: GameData/RP-0/Avionics.cfg (hunk 1 above)
Comment (Asgarmagol) [inline GameData/RP-0/Avionics.cfg]: I was only looking for Block D in combination with Proton. 20t is the max payload of the Proton. Block D is part of the Payload. so it is correct for this usage.

**Pull request merged**